### PR TITLE
Don't show unpublished posts as pickup

### PR DESCRIPTION
**Pull request by tricknotes**

### CODE REVIEW

File: app/views/editor/pickup_posts/_form.html.haml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 = bootstrap_form_for [:editor, @pickup_post], layout: :horizontal, label_col: "col-sm-2", control_col: "col-sm-10" do |f|
-  = f.collection_select :post_id, current_site.posts, :id, :title, {prompt: '選択してください'}, {class: 'form-control select2', "data-select2-allow-clear": false}
+  = f.collection_select :post_id, @candidate_posts, :id, :title_with_public_id, {prompt: '選択してください'}, {class: 'form-control select2', "data-select2-allow-clear": false}
```
Comment (tricknotes) [inline app/views/editor/pickup_posts/_form.html.haml:2]: @ryota-sasabe 
I think `Post#public_id` is worthful here.
How about this?

Comment (ryota-sasabe) [reply app/views/editor/pickup_posts/_form.html.haml:2]: It's Good !

File: app/views/application/_pickup_posts.html.haml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 - if pickup_posts.exists?
   %h2.subheading.subheading--aside.subheading--press-articles.subheading--press-articles--pickup ピックアップ記事
-  %ul
-    - pickup_posts.includes(post: [:categories]).ordered.each do |pickup_post|
+  %ul.articles--pickup
```
Comment (mtsmfm) [inline app/views/application/_pickup_posts.html.haml]: `.pickup-articles` or `.articles.articles--pickup`

You shouldn't use modifier only.
Modifier is just like a "mix-in".

Comment (tricknotes) [reply app/views/application/_pickup_posts.html.haml]: nrhd!

File: app/views/application/_pickup_posts.html.haml [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
+    - pickup_posts.includes(post: [:categories]).published.ordered.each do |pickup_post|
```
Comment (mtsmfm) [inline app/views/application/_pickup_posts.html.haml]: test

**Review by mtsmfm — APPROVED** [review]

**Comment by tricknotes** [comment]
🚀

**Pull request merged**